src/widget/tool/screenshotgrabber.cpp

```diff
@@ -181,14 +181,20 @@ void ScreenshotGrabber::chooseHelperTooltipText(QRect rect)
         useRegionSelectedTooltip();
 }
 
+/**
+ * @internal
+ *
+ * Align the tooltip centred at top of screen with the mouse cursor.
+ */
 void ScreenshotGrabber::adjustTooltipPosition()
 {
-    QRectF size = this->helperToolbox->childrenBoundingRect();
-    QRect screenRect = QApplication::desktop()->screen()->rect();
-
-    // Align the toolbox center-top.
-    helperToolbox->setX(screenRect.x() + (screenRect.width() - size.width() + size.x()) / 2);
-    helperToolbox->setY(screenRect.y());
+    QRect recGL = QGuiApplication::primaryScreen()->virtualGeometry();
+    QRect rec = qApp->desktop()->availableGeometry(QCursor::pos());
```

Comment:
I mean this line. replace with `QRect rec = qApp->desktop()->screenGeometry(QCursor::pos());`